src/com/lftechnology/java/training/sanish/generics/GenericTest.java

```diff
@@ -0,0 +1,138 @@
+package com.lftechnology.java.training.sanish.generics;
+
+import com.lftechnology.java.training.sanish.generics.customer.BankCustomer;
+import com.lftechnology.java.training.sanish.generics.customer.Customer;
+import com.lftechnology.java.training.sanish.generics.customer.RestaurantCustomer;
+import com.lftechnology.java.training.sanish.generics.product.Computer;
+import com.lftechnology.java.training.sanish.generics.product.Mouse;
+import com.lftechnology.java.training.sanish.generics.product.Product;
+import com.lftechnology.java.training.sanish.generics.vehicle.Car;
+import com.lftechnology.java.training.sanish.generics.vehicle.VehicleList;
+import com.lftechnology.java.training.sanish.generics.vehicle.VehicleListImpl;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Generic class, interface & function test
+ *
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class GenericTest {
+    private static final Logger LOGGER = Logger.getLogger(VehicleListImpl.class.getName());
+
+    private GenericTest() {
+
+    }
+
+    public static void main(String[] args) {
+        // add two number using generic function
+        addIntegerNumbers(2, 5);
+        addDoubleNumbers(3.5, 7.75);
+
+        // add / remove vehicle list of diffenrent type with generic class
+        workOnCarList();
+        workOnBikeList();
+
+        //Instantiate an object of Product using different data types.
+        Product computerProduct = new Product(new Computer("Dell", "Intel® Core™ i5-3337U CPU @ 1.80GHz × 4 ", "7.7 GiB", "500.0 GB"));
+        computerProduct.displayProductInfo();
+
+        Product mouseProduct = new Product(new Mouse("Delux", "Optical mouse"));
+        mouseProduct.displayProductInfo();
+
+        // display customers information using wildcard function
+        List<BankCustomer> bankCustomers = new ArrayList<BankCustomer>();
+        bankCustomers.add(new BankCustomer(1, "Sanish", "9841115968", 145258696, "Global", "Saving"));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "9841115968" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)